### ISSUE TITLE
Clean install throws "unexpected token error" everytime

### ISSUE THREAD

**Issue by samayo** [rchipka/node-osmosis#156]
Sorry if this is too noob, but I installed node/npm and osmosis, all went fine and dumped a working example inside `index.js` 
```js
var osmosis = require('osmosis');

osmosis
.get('www.craigslist.org/about/sites')
.find('h1 + div a')
.set('location')
.follow('@href')
.find('header + nav + div + div li > a')
.set('category')
.follow('@href')
.paginate('.totallink + a.button.next:first')
.find('p > a')
.follow('@href')
.set({
    'title':        'section > h2',
    'description':  '#postingbody',
    'subcategory':  'div.breadbox > span[4]',
    'date':         'time@datetime',
    'latitude':     '#map@data-latitude',
    'longitude':    '#map@data-longitude',
    'images':       ['img@src'],
    'id': 'p.postinginfo[0]'
})
.data(function(listing) {
    // do something with listing data
    console.log(listing);
})
.log(console.log)
.error(console.log)
.debug(console.log)
```
But when I run `npm index.js` I get this error 
```bash
C:\Users\..\node_modules\osmosis\lib\commands\request.js:22
}
^
SyntaxError: Unexpected token }
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at C:\Users\..\node_modules\osmosis\lib\Command.js:542:19
    at Array.forEach (native)
```

Line 22 is the first line in this 
```bash
   if (Object.getPrototypeOf(parent) === Command.prototype) {  
       // parent is a Command
       this.prev = parent;
       Object.defineProperty(this, 'instance', {
           get: Command.prototype.getInstance,
           set: Command.prototype.setInstance
       });
   } else if (parent !== undefined) {
```

Is there something i am doing wrong?

**Comment by Connorelsea** [comment]
I am getting this as well.

**Comment by queenvictoria** [comment]
I rolled back and it works
```
$ npm install osmosis@1.1.2 --save
```

**Comment by rchipka** [comment]
Looks like some odd changes got into the NPM release but not the master branch.

**Comment by rchipka** [comment]
Published v1.1.4

**Comment by queenvictoria** [comment]
Works for me thanks!
